nemo/collections/asr/parts/submodules/rnnt_greedy_decoding.py

```diff
@@ -1079,9 +1301,13 @@ def _get_initial_states(self, batchsize):
 class GreedyRNNTInferConfig:
     max_symbols_per_step: Optional[int] = 10
     preserve_alignments: bool = False
+    preserve_frame_confidence: bool = False
+    confidence_method_cfg: Optional[ConfidenceMethodConfig] = None
```

Comment:
Why are the configs None? Make it easier for the user by providing a default, it wont be used anyway due to preserve_frame_confidence=False.